app/main.py

```diff
@@ -1,8 +1,47 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line